web/cypress/integration/Trace/CreateAssertion.spec.ts

```diff
@@ -0,0 +1,91 @@
+import { createTest, deleteTest, testId } from "../utils/common";
+
+describe('Create Assertion', () => {
+  before(() => {
+    createTest();
+  });
+
+  after(() => {
+    deleteTest();
+  });
+
+  it('should create a basic assertion', () => {
+    cy.get(`[data-cy=test-url-${testId}]`).first().click();
+    cy.location('href').should('match', /\/test\/.*/i);
+
+    cy.get(`[data-cy^=test-run-result-]`).first().click();
+    cy.location('href').should('match', /resultId=.*/i);
+
+    cy.wait(5000);
```

Comment:
Can't you replace this with something like:

```js
cy.get('[data-cy=add-assertion-button]', { timeout: 10000 }).should('be.visible');
```

PS: I didn't test it, but I'm almost sure that would work.